python/python/tests/test_vector_index.py

```diff
@@ -124,6 +127,26 @@ def test_ann(indexed_dataset):
     print(run(indexed_dataset))
 
 
+@pytest.mark.skipif(
+    (platform.system() == "Darwin") and (platform.machine() != "arm64"),
```

Comment:
the intel mac should fallback to the AVX version? not sure why need to skip on mac GHA

Reply:
we added this awhile back because the index related tests on intel mac runners would take forever on github